baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -63,14 +64,17 @@ def __init__(
         queues: Sequence[kombu.Queue],
         work_queue: WorkQueue,
         serializer: Optional[KombuSerializer] = None,
+        **kwargs: Any,
     ):
         self.connection = connection
         self.queues = queues
         self.work_queue = work_queue
         self.serializer = serializer
+        self.kwargs = kwargs
 
     def get_consumers(self, Consumer: kombu.Consumer, channel: Channel) -> Sequence[kombu.Consumer]:
         args = dict(queues=self.queues, on_message=self.work_queue.put)
+        args.update(self.kwargs)
```

Comment:
would adding `, **self.kwargs)` to the `dict()` call above do this in fewer words?

Reply:
That'll give you an error if "kwargs" has a value that is already being passed in explicitly while this overrides the value.

Reply:
Both are valid IMO, just different.

Reply:
If repeat kwargs cause errors, is that going to be a problem?